libcxx/include/__tree

```diff
@@ -1009,9 +1017,27 @@ public:
   __insert_node_at(__end_node_pointer __parent, __node_base_pointer& __child, __node_base_pointer __new_node) _NOEXCEPT;
 
   template <class _Key>
-  _LIBCPP_HIDE_FROM_ABI iterator find(const _Key& __v);
+  _LIBCPP_HIDE_FROM_ABI iterator find(__multi_tag, const _Key& __v);
+  template <class _Key>
+  _LIBCPP_HIDE_FROM_ABI const_iterator find(__multi_tag, const _Key& __v) const;
+
+  template <class _Key>
+  _LIBCPP_HIDE_FROM_ABI iterator find(__unique_tag, const _Key& __key) {
+    __end_node_pointer __parent;
+    __node_base_pointer __match = __find_equal(__parent, __key);
```

Comment:
Per your observation just now, it looks like it should be conforming to just use `__find_equal` in `find` even for multimap. There's no standard guarantee that we return the first element with a given key, users must use `lower_bound` for that.